src/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from './parseAge'
```

Comment:
replace with https://github.com/DeFiCh/scan/blob/b93efa3cec97a469ad5ce499fa6b8ae75ff822ee/src/pages/blocks/index.tsx#L8

Reply:
This utility function is deleted.